core/tests/protractor_utils/ExplorationEditorFeedbackTab.js

```diff
@@ -60,12 +61,14 @@ var ExplorationEditorFeedbackTab = function() {
       `${suggestionRowClassName} ${feedbackSubjectClassName}`,
       suggestionDescription));
     expect(await matchingRow.isDisplayed()).toBe(true);
```

Comment:
Could you remove this line too? It isn't needed since `action.click` will fail if `matchingRow` is not visible